js/ui/date_range_box.js

```diff
@@ -0,0 +1,33 @@
+import registerComponent from '../core/component_registrator';
```

Comment:
May be we need create a folder for date_range_box component and move this code here? In this file we last import and export only

```
import DateRangeBox from './date_range_box/ui.date_range_box';
export default DateRangeBox;
```